doc/api/corepack.md

```diff
@@ -16,6 +16,6 @@ Documentation for this tool can be found on the [Corepack repository][].
 
 Despite Corepack being distributed with default installs of Node.js, the package
 managers managed by Corepack are not part of the Node.js distribution, and
-Corepack itself will no longer be distributed with future versions of Node.js.
+Corepack itself will no longer be distributed with Node.js v25+.
```

Comment:
Maybe `v25.x and above`, instead of the short-hand form `v25+` ?

Reply:
You're right. It's better to be clear
```suggestion
Corepack itself will no longer be distributed with Node.js v25.x and above.
```

Reply:
As per https://github.com/nodejs/Release, we don't use `vN.x`, but directly use `N.x`

But when Node.js is in the prefix, we don't use `v` or `.x`.

Reply:
I'll just remove the v as of now, and will wait for input from other collaborators
```suggestion
Corepack itself will no longer be distributed with Node.js 25+.
```

I verified that official announcements use number directly after `Node.js`
https://nodejs.org/en/blog/announcements